app/assets/javascripts/global.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * global.js
+
+ * Include in this file any javascript meant be run globally. That is, for any page on
+ * document ready.
+ *
+ * This file models the new (as of 2017) preferred pattern we'd like to use for Javascript.
+ * It follows jQuery documentation here:
+ *
+ * https://learn.jquery.com/using-jquery-core/document-ready/
+ *
+ * For more information on best practices, see: https://kb-smc.chapman.edu/?p=1796
+ *
+ * See also app/assets/javascripts/cascade/main.js
+ */
+var globalLoader = (function() {
+
+    // Constants
+
+    // Global Attrs: undefined values will be set on document ready.
+    var $omninav = undefined;
+    var omninavPresent = undefined;
+    var omninavHeight = undefined;
+
+    // Public Methods
+    var initOnDocumentReady = function() {
+        // Set undefined globals.
+        $omninav = $('#cu_nav');
+        omninavPresent = $omninav.length > 0;
+        omninavHeight = omninavPresent ? $omninav.outerHeight() : null;
+
+        // Init events.
+        repositionAnchorLinksToOffsetOmniNav();
+    }
+
+    // Private Methods
+    var repositionAnchorLinksToOffsetOmniNav = function() {
+        // Only apply this when omninav is present.
+        if ( ! omninavPresent ) {
+            return false;
+        }
+
+        // Unfortunately, there isn't a uniform identifier that can be referenced throughout
+        // the site at this time. So we have to cover a few cases.
+        var $wysiwygLinks = $('h3.anchorLinks a, a[href="#top"]');
```

Comment:
Currently, we cover all the cases that have been brought to our attention. This selector was brought over from your last story. I added a new one for A-Z links below. (Personnel widget, below, needed to be handled differently.) 

I'd prefer not to overgeneralize because I'm not sure what unintended side effects may be.